packages/frontend/src/scripts/charts/styles.ts

```diff
@@ -73,6 +74,10 @@ export const POINT_CLASS_NAMES: PointClassNames = {
     type: 'div',
     className: 'size-2 rotate-45 border-2 border-green-500 bg-green-700',
   },
+  incident: {
+    type: 'div',
+    className: 'size-2 rotate-45 border-2 border-red-700 bg-red-800',
+  },
```

Comment:
its not a triangle, is it ok?

Reply:
Triangles look weird on the chart, that's why I left it like this.